utils/hono-influxdb-connector/pushImage.sh

```diff
@@ -17,9 +17,9 @@ SCRIPTPATH=$(dirname "$(readlink -f "$0")")
 DOCKER_REGISTRY_SERVER=$1
 DOCKER_REGISTRY_USERNAME=$2
 DOCKER_REGISTRY_PASSWORD=$3
+VERSION=${4:-"0.1.0"}
 
 DOCKER_IMAGE_NAME="hono-influxdb-connector"
-VERSION="$(gradle properties -q | grep 'version:' | awk '{print $2}')"
```

Comment:
This change triggered my curiosity: How/where do we want to configure version numbers? And how do we want to increment them?

The removed line uses the version information in gradle as a "single" source of versioning information. Every `build.gradle` file in this Git repo has this information, though, at times as `bootJar.version` instead of `version`, so there is redundancy.

It would be possible to remove that information from the `build.gradle` files - especially if it is not used for the Docker images. Then, the `$VERSION` passed to the scripts could be used to set the version of the Docker images and the Git repository could remain clear of versions. I.e. also clear of the default versions added to the Bash scripts.

What do you think? Did you have a particular use case in mind when adding the version argument to the scripts besides the default versions given in the scripts?

Reply:
You are right, the way we manage the version number is not fixed yet and open for discussion. 

The use case I had in mind was to be able to use and deliver the pushImage.sh script as part of the release archive. Since the build.gradle file will not be part of the release archive we can not use it as source for the version number. My idea was further to later create a pushAllImages.sh which calls the respective pushImage.sh scripts of the different subprojects (e.g HonoInfluxDbConnector) and could then specify the version number. 

In general, I would prefer to use the same version number for all subprojects in accordance to the version number of the current release which would be 0.1 at the moment. If we agree on that it would be more convenient to store the version only in one central location. This could be  something like a version file from which all other scripts read the current version. 
Because of that I agree that we then could remove the version number from the build.gradle file of the sub-projects.

Reply:
Thanks for elaborating! I agree with you: all parts of the release should have the same version number.

What's your preference: Would you like to update this PR with the changes you proposed or should I merge it and create a follow-up issue? I'm fine with any of the options.

Reply:
I prefere the solution with the merge and follow-up issue.